package.json

```diff
@@ -3,11 +3,13 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "bootstrap": "^4.5.3",
```

Comment:
Good question! It's necessary as they state in the documentation `Because React-Bootstrap doesn't depend on a very precise version of Bootstrap, we don't ship with any included css`